apps/condo/domains/news/schema/NewsItemScope.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Generated by `createschema news.NewsItemScope 'newsItem:Relationship:NewsItem:CASCADE; property:Relationship:Property:CASCADE; unitType:Select:get,from,constant,unit_types; unitName:Text'`
+ */
+
+const { Text, Relationship,  Select } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/news/access/NewsItemScope')
+const { UNIT_TYPES } = require('@condo/domains/property/constants/common')
+
+const NewsItemScope = new GQLListSchema('NewsItemScope', {
+    schemaDoc: 'Which residents can see the particular news item',
+    fields: {
+
+        newsItem: {
```

Comment:
Is this a unique field? One news item may have several scopes?

Reply:
Yes, each news item may be shared to residents using several scopes